client-js/app/elements/labrad-plot.ts

```diff
@@ -511,12 +602,37 @@ export class Plot extends polymer.Base {
         p = this;
 
     // adjust size, position, and color of each data rect
-    this.svg.selectAll('rect.data')
-        .attr('x', (d) => p.xScale(d[0]))
-        .attr('y', (d) => p.yScale(p.yNext[d[1]]))
-        .attr('width', (d) => p.xScale(p.xNext[d[0]]) - p.xScale(d[0]))
-        .attr('height', (d) => Math.abs(p.yScale(p.yNext[d[1]]) - p.yScale(d[1])))
-        .style('fill', (d) => getColor(d[2], zMin, zMax));
+    switch (this.drawMode2D) {
```

Comment:
I haven't looked at this in a long time so I'm not sure where it is, but the rightmost border doesn't show up as black on the plots.